test/brsTypes/Int64.test.js

```diff
@@ -11,18 +11,18 @@ const {
 const Long = require("long");
 
 describe("Int64", () => {
-    it("rounds initial values to integers", () => {
+    it("truncates initial values to integers", () => {
         let three = new Int64(3.4);
         let four = new Int64(3.5);
```

Comment:
Would you mind renaming the variable `four` to `threePointFive` or something?   Or drop the intermediate values entirely, since both `3.4` and `3.5` should convert to `3` either way!  The meaning of the variable names is no longer useful, since they have the same resulting value 😄